src/main/java/mongodb/Main.java

```diff
@@ -0,0 +1,220 @@
+    MongoClientSettings settings = MongoClientSettings.builder()
+            .codecRegistry(pojoCodecRegistry)
+            .applyConnectionString(mongoUri).build();
+    MongoClient mongoClient = MongoClients.create(settings);
+
+    // MongoDatabase defines a connection to a specific MongoDB database
+    MongoDatabase database = mongoClient.getDatabase(dbName);
+    // MongoCollection defines a connection to a specific collection of documents in a specific database
+    MongoCollection<Recipe> collection = database.getCollection(collectionName, Recipe.class);
+
+    /*      *** INSERT DOCUMENTS ***
+     *
+     * You can insert individual documents using collection.insert().
+     * In this example, we're going to create 4 documents and then
+     * insert them all in one call with insertMany().
+     */
+
+    List<Recipe> recipes = getRecipes();
+
+    try {
+      InsertManyResult result = collection.insertMany(recipes);
+      System.out.println("Inserted document ids: " + result.getInsertedIds());
+    } catch (MongoException me) {
+      System.err.println("Unable to insert any driver documents due to an error: " + me);
+    }
+
+    /*      *** FIND DOCUMENTS ***
+     *
+     * Now that we have data in Atlas, we can read it. To retrieve all of
+     * the data in a collection, we call find() with an empty filter.
+     */
+
+    try (MongoCursor<Recipe> cursor = collection.find().iterator()) {
+      while (cursor.hasNext()) {
+        System.out.println(cursor.next());
+      }
+    } catch (MongoException me) {
+      System.err.println("Could not find documents due to an error: " + me);
+    }
+
+    // We can also find a single document. Let's find the first document
+    // that has the string "potato" in the Ingredients list. We
+    // use the Filters.eq() method to search for any values in any
+    // ingredients list that match the string "potato":
+
+    Bson findPotato = eq("ingredients", "potato");
+    try {
+      Recipe firstPotato = collection.find(findPotato).first();
+      if (firstPotato == null) {
+        System.out.println("Didn't find any recipes containing 'potato' as an ingredient.");
+      } else {
+        System.out.println(firstPotato);
+      }
+    } catch (MongoException me) {
+      System.err.println("Could not find single document due to an error: " + me);
+    }
+
+    /*      *** UPDATE A DOCUMENT ***
+     *
+     * You can update a single document or multiple documents in a single call.
+     *
+     * Here we update the PrepTimeInMinutes value on the document we
+     * just found.
+     */
+    Bson updateFilter = Updates.set("prepTimeInMinutes", 72);
+
+    // The following FindOneAndUpdateOptions specify that we want it to return
+    // the *updated* document to us. By default, we get the document as it was *before*
+    // the update.
+    FindOneAndUpdateOptions options = new FindOneAndUpdateOptions();
+    options.returnDocument(ReturnDocument.AFTER);
```

Comment:
Question:
Can you do:
```
FindOneAndUpdateOptions options = new FindOneAndUpdateOptions().returnDocument(ReturnDocument.AFTER);
```
Since `returnDocument()` returns `this`, and if so, would that be more concise?